### PR TITLE
New Faction/Trader Accesses

### PR DESCRIPTION
**Pull request by Blackern5000**
I made a bunch of special accesses for the traders and each faction.
There's a regular access and a command access for each faction, including traders, and then a single hostile access for independent ships so they can't be boarded by the players.
I added two trader jobs (Station Administrator and Trade Technician) who have the new trader accesses but I don't know how to make them spawn instead of the QM at trade stations so I'll let you do that.
I made eight IncSek airlocks, eight Coregov airlocks, four trader airlocks and two hostile airlocks, these should make breaking into ships with doors harder.

(Don't mind me accidentally adding the musca at the start I got my branches mixed up.)

### CODE REVIEW

**Comment by Blackern5000** [comment]
unfortunately no longer contains trader jobs as advertised due to it not working for some reason

File: Resources/Prototypes/_FTL/Access/ekrixi.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
+- type: accessLevel
+  id: Coregov
+  name: id-card-access-level-coregov
```
Comment (Just-a-Unity-Dev) [inline Resources/Prototypes/_FTL/Access/ekrixi.yml:3]: This locale doesn't exist?

Comment (Just-a-Unity-Dev) [reply Resources/Prototypes/_FTL/Access/ekrixi.yml:3]: Same goes for the rest of the access levels

**Review by Just-a-Unity-Dev — CHANGES_REQUESTED** [review]
Split the accesses from a single `ekrixi.yml` file to their own faction-specific files

**Comment by Blackern5000** [comment]
> > Split the accesses from a single `ekrixi.yml` file to their own faction-specific files
> 
> Do you want this done with the airlocks file too?

I did it anyway

**Comment by Just-a-Unity-Dev** [comment]
sgtm

**Pull request merged**